src/ontology/mondo-edit.obo

```diff
@@ -305096,6 +305096,7 @@ xref: OMIMPS:114580 {source="MONDO:equivalentTo"}
 xref: Orphanet:1334 {source="DOID:2058", source="MONDO:equivalentTo"}
 xref: SCTID:234568006 {source="DOID:2058", source="MONDO:equivalentTo"}
 xref: UMLS:C0006845 {source="Orphanet:1334", source="DOID:2058", source="MONDO:equivalentTo", source="NCIT:C34444", source="Orphanet:1334/e"}
+is_a: MONDO:0003778 {source="https://orcid.org/0000-0002-0736-9199"} ! inborn error of immunity
```

Comment:
@sabrinatoro could you please add a link to the GitHub URL too? thanks!

Reply:
done